src/vscode-bicep/src/commands/pasteAsBicep.ts

```diff
@@ -0,0 +1,279 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import {
+  env,
+  TextDocumentChangeEvent,
+  TextDocumentChangeReason,
+  Uri,
+  window,
+  workspace,
+  WorkspaceEdit,
+  Range,
+  MessageItem,
+  ConfigurationTarget,
+} from "vscode";
+import { Command } from "./types";
+import { LanguageClient } from "vscode-languageclient/node";
+import { findOrCreateActiveBicepFile } from "./findOrCreateActiveBicepFile";
+import {
+  callWithTelemetryAndErrorHandling,
+  IActionContext,
+} from "@microsoft/vscode-azext-utils";
+import { Disposable } from "../utils";
+import { areEqualIgnoringWhitespace } from "../utils/areEqualIgnoringWhitespace";
+import { getTextAfterFormattingChanges } from "../utils/getTextAfterFormattingChanges";
+import { bicepConfigurationKeys, bicepLanguageId } from "../language/constants";
+import {
+  BicepDecompileForPasteCommandParams,
+  BicepDecompileForPasteCommandResult,
+} from "../language";
+import { OutputChannelManager } from "../utils/OutputChannelManager";
+import { getBicepConfiguration } from "../language/getBicepConfiguration";
+import { SuppressedWarningsManager } from "./SuppressedWarningsManager";
+
+export class PasteAsBicepCommand implements Command {
+  public static readonly id = "bicep.pasteAsBicep";
```

Comment:
Can we just do?
```
public readonly id = "bicep.pasteAsBicep";
```

Reply:
153b64b8f1c67fed1835d425fc2b35c630afff6a